src/index.ts

```diff
@@ -13,14 +13,39 @@
 // limitations under the License.
 
 export * as Errors from './errors';
-export {Manifest, ReleaserConfig, ManifestOptions} from './manifest';
 export {
+  Manifest,
+  ReleaserConfig,
+  ManifestOptions,
+  PluginType,
+} from './manifest';
+export {Commit, ConventionalCommit} from './commit';
+export {BaseStrategyOptions, BuildUpdatesOptions} from './strategies/base';
+export {
+  AnyStrategyOptions,
+  ChangelogNotesBuilder,
+  ChangelogNotesFactoryOptions,
+  ChangelogNotesType,
+  PluginBuilder,
+  PluginFactoryOptions,
+  ReleaseBuilder,
   ReleaseType,
-  getReleaserTypes,
+  VersioningStrategyBuilder,
+  VersioningStrategyFactoryOptions,
   VersioningStrategyType,
-  getVersioningStrategyTypes,
-  ChangelogNotesType,
   getChangelogTypes,
+  getPluginTypes,
+  getReleaserTypes,
+  getVersioningStrategyTypes,
+  registerChangelogNotes,
+  registerPlugin,
```

Comment:
A couple of these exports don't seem to be from the correct source file.

Reply:
I'll take a look.

Reply:
Ok, so its matter of opinion, IDEA always suggests to use shortest import path possible (thus in case of a re-export in factory it suggests to use that). I actually agree with that, since its always better to use higher-level abstraction, even with node modules.
I'll modify it to always use source file directly.